rero_ils/dojson/utils.py

```diff
@@ -293,6 +293,8 @@
 _CONTRIBUTION_TAGS = ['100', '600', '610', '611', '630', '650', '651',
                              '655', '700', '710', '711']
 
+_SPECIAL_CHARACTERS = ['\u0098', '\u009C']
```

Comment:
Why a global variable? I would put this a default value of a parameter of the `remove_special_characters` function:
```python
def remove_special_characters(value, chars=['\u0098', '\u009C']):
...
```